frontend/packages/console-shared/src/components/synced-editor/index.tsx

```diff
@@ -59,6 +59,7 @@ export const SyncedEditor: React.FC<SyncedEditorProps> = ({
       .then((js) => {
         setSwitchError(undefined);
         handleFormDataChange(js);
+        setYAML('');
```

Comment:
This feels like you're doing something to help yourself out -- this isn't required in other spots in the app, so we should avoid special use-cases polluting common components. 

Please remove and see my next comment.

Reply:
> This feels like you're doing something to help yourself out -- this isn't required in other spots in the app, so we should avoid special use-cases polluting common components.
> 
> Please remove and see my next comment.

@andrewballantyne Added this to clear the existing yaml so that the editor would use the updated yaml on save. Without it  the first update works, but  subsequent updates throws error `"Operation cannot be fulfilled on poddisruptionbudgets.policy \"reactapp\": the object has been modified; please apply your changes to the latest version and try again"
metadata: {}
reason: "Conflict"
status: "Failure"` .  The Reload btn doesn't work in this case.

I tried assigning the `yamlContext`,  it works but the user would have to click the reload button for subsequent updates.
` setYAML(yamlContext ? safeJSToYAML(yamlContext) : newYAML);`

Not sure what would be the best way to fix without tweaking the common component. 
Please take a look when you get a chance.